frappe/public/scss/common/indicator.scss

```diff
@@ -10,7 +10,7 @@
 @mixin indicator-color($color) {
 	&::before,
 	&::after {
-		background: var(--text-on-#{$color});
+		background: #{$color};
```

Comment:
Now the color is coming from the bootstrap CSS colors. Which is different from our shades of color but it still looks fine. Will approve it.
Can you add some before-after screenshots both in light mode and dark mode